### PR TITLE
br: add restriction for incremental restore

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? (Required)

Add some restriction of incremental restore

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/17579
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/br-incremental-guide.md [hunk 1]
```diff
@@ -11,6 +11,12 @@ Incremental data of a TiDB cluster is differentiated data between the starting s
 >
 > Development for this feature has stopped. It is recommended that you use [log backup and PITR](/br/br-pitr-guide.md) as an alternative.
 
+## Limitations
+
+Because restoring the incremental backup relies on the snapshot of the database table at the backup time point to filter incremental DDL statements, the tables deleted during the incremental backup process might still exist after the data restore and need to be manually deleted.
+
+The incremental backup does not support batch renaming of incremental DDL statements. If batch renaming of tables occurs within the selected incremental backup range, the data restore might fail. It is recommended to perform a full backup after batch renaming the incremental DDL statements, and use the latest full backup to replace the incremental data during restore.
```
Comment (BornChanger) [inline br/br-incremental-guide.md:18]:
```suggestion
The incremental backup does not support batch table renaming statements. If batch renaming of tables occurs within the selected incremental backup range, the data restore might fail. It is recommended to perform a full backup after batch table renaming statements, and use the latest full backup to replace the incremental data during restore.
```

**Comment by lilin90** [comment]
@BornChanger PTAL

File: br/br-incremental-guide.md [hunk 2]
```diff
@@ -11,6 +11,12 @@ Incremental data of a TiDB cluster is differentiated data between the starting s
 >
 > Development for this feature has stopped. It is recommended that you use [log backup and PITR](/br/br-pitr-guide.md) as an alternative.
 
+## Limitations
+
+Because restoring the incremental backup relies on the snapshot of the database table at the backup time point to filter incremental DDL statements, the tables deleted during the incremental backup process might still exist after the data restore and need to be manually deleted.
+
+The incremental backup does not support batch renaming of tables. If batch renaming of tables occurs during the range of the incremental backup, the data restore might fail. It is recommended to perform a full backup after batch renaming tables, and use the latest full backup to replace the incremental data during restore.
```
Comment (BornChanger) [inline br/br-incremental-guide.md:18]:
```suggestion
The incremental backup does not support batch renaming of tables. If batch renaming of tables occurs during the incremental backup process, the data restore might fail. It is recommended to perform a full backup after batch renaming tables, and use the latest full backup to replace the incremental data during restore.
```

**Review by BornChanger — APPROVED** [review]

**Comment by lilin90** [comment]
After confirming with @BornChanger, we won't apply changes in this PR to v6.1 docs.

**Pull request merged**